subgraph-js/README.md

```diff
@@ -7,8 +7,10 @@ For complete documentation, see the [Apollo Subgraph API reference](https://www.
 ## Usage
 
 ```js
-const { ApolloServer, gql } = require("apollo-server");
-const { buildSubgraphSchema } = require("@apollo/subgraph");
+import { ApolloServer } = from '@apollo/server';
+import { startStandaloneServer } = from '@apollo/server/standalone';
+import { gql } = from 'graphql-tag';
+import { buildSubgraphSchema } = from '@apollo/subgraph';
```

Comment:
```suggestion
import { ApolloServer } from '@apollo/server';
import { startStandaloneServer } from '@apollo/server/standalone';
import { gql } from 'graphql-tag';
import { buildSubgraphSchema } from '@apollo/subgraph';
```

Reply:
That's embarrassing! Thank you! Been looking at too many imports 🤦‍♀️

Reply:
Easy find and replace mistake!